docs/integration/KafkaIntegration.md

```diff
@@ -16,24 +16,24 @@ If you want to use Flink engine, this is also recommended:
 
 ## Sources and sinks
 
-Kafka topics are native streaming data input to Nussknacker and the native output where results of Nussknacker scenarios processing are placed. In Nussknacker terminology input topics are called sources, output topics are called sinks. This section provides important details of Nussknacker's integration with Kafka and schema registry.
+Kafka topics are native streaming data input to Nussknacker and the native output where results of Nussknacker scenarios processing are placed. In Nussknacker terminology input topics are called sources, output topics are called sinks. This section provides important details of Nussknacker's integration with Kafka and Schema Registry.
 
-## Schema registry integration
+## Schema Registry integration
 
-Nussknacker integrates with a schema registry. It is the source of topics available to choose from in sources and sinks. It also allows Nussknacker to provide syntax suggestions and validation. Nussknacker assumes that for the topic `topic-name` a schema `topic-name-value` and optionally `topic-name-key` (for the Kafka topic key) will be defined in the schema registry.
+Nussknacker integrates with Confluent Schema Registry. It is the source of topics available to choose from in Kafka sources and sinks. It also allows Nussknacker to provide syntax suggestions and validation. Nussknacker assumes that for the topic `topic-name` a schema `topic-name-value` and optionally `topic-name-key` (for the Kafka topic key) will be defined in the schema registry.
```

Comment:
Not necessarily Confluent, let's skip it

Reply:
Fixed